### PR TITLE
feat(deps): remove dependency on gen_random_uuid()/pgcrypto

### LINKED ISSUE #249: pgcrypto not necessary in PostgreSQL 13+?
Correct me if I'm wrong, but pgcrypto only appears to be installed in order to use `gen_random_uuid()`. That function [is available without the extension](https://www.postgresql.org/docs/current/functions-uuid.html) in Postgres 13+. Is it feasible to make the installation of pgcrypto conditional on version so it can be avoided if unnecessary?

### PR DESCRIPTION
**Pull request by noinkling**
## Description

Alternative to/supersedes #251, as per [comment](https://github.com/graphile/worker/pull/251#issuecomment-1106608649). Fixes #249.

I only edited the existing migrations the minimum amount to prevent them from failing. It's probably possible to remove large parts of the old cruft, but as you are no doubt aware, the idea of migrations is not to edit them later (ideally), and you would have to be more careful not to break users upgrading from an old version, so I didn't attempt that.

## Performance impact

unknown

## Security impact

unknown

## Checklist

- [x] My code matches the project's code style and `yarn lint:fix` passes.
- [x] ~~I've added tests for the new feature, and~~ `yarn test` passes.
- [x] I have detailed the new feature in the relevant documentation.
- [x] I have added this feature to 'Pending' in the `RELEASE_NOTES.md` file (if one exists).
- [x] If this is a breaking change I've explained why.

### CODE REVIEW

File: sql/000001.sql [hunk 1]
```diff
@@ -79,7 +79,7 @@ create trigger _900_notify_worker after insert on :GRAPHILE_WORKER_SCHEMA.jobs f
 create function :GRAPHILE_WORKER_SCHEMA.add_job(
   identifier text,
   payload json = '{}',
-  queue_name text = public.gen_random_uuid()::text,
+  queue_name text = 'PLACEHOLDER',
```
Comment (noinkling) [inline sql/000001.sql:82]: This feels weird, but technically at this point in time the column is non-nullable so I used a string that makes it clear that it isn't actually going to be used.

Comment (benjie) [reply sql/000001.sql:82]: Use `null /* was gen_random_uuid(), but later removed dependency */`

File: README.md [hunk 2]
```diff
@@ -208,11 +208,11 @@ http://discord.gg/graphile
...
 PostgreSQL 10+\* and Node 10+\*.
 
-If your database doesn't already include the `pgcrypto` extension we'll
-automatically install it into the public schema for you. If the extension is
-installed in a different schema (unlikely) you may face issues. Making alias
-functions in the public schema, should solve this issue (see issue
-[#43](https://github.com/graphile/worker/issues/43) for an example).
+_Note: `graphile-worker` versions 0.12.2 and earlier installed the `pgcrypto`
+extension into the public schema of your database (if it wasn't already
+installed). As of `graphile-worker` version _\<pending\>_ we no longer use
```
Comment (benjie) [inline README.md:213]:
```suggestion
installed). As of `graphile-worker` version 0.13.0 we no longer use
```

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,5 +1,12 @@
 # Release notes
 
+### Pending
+
+- Remove dependency on `pgcrypto` database extension
+  - If you have a pre-existing installation and wish to uninstall `pgcrypto` you
+    will need to do so manually. This can be done by running
+    `DROP EXTENSION pgcrypto;` _after_ updating to the latest schema.
```
Comment (benjie) [inline RELEASE_NOTES.md:8]:
```suggestion
    `DROP EXTENSION pgcrypto;` _after_ updating to the latest schema.
- jobs.queue_name no longer has a default (this is only relevant to people inserting to the table directly, which is not recommended - use the add_job helper)
```

**Review by benjie — CHANGES_REQUESTED** [review]
Looking good!

**Review by benjie — APPROVED** [review]
Excellent; this will make it even easier for people to adopt Graphile Worker - thanks! :raised_hands:

**Comment by benjie** [comment]
Released in v0.13.0 :raised_hands:

**Pull request merged**